pkg/driver_sx126x/contrib/driver_sx126x_hal.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_driver_sx126x
+ * @{
+ *
+ * @file
+ * @brief       HAL implementation for the SX1261/2 LoRa radio driver
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include "fmt.h"
+#include "ztimer.h"
+
+#include "periph/gpio.h"
+#include "periph/spi.h"
+
+#include "sx126x.h"
+#include "sx126x_hal.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+#define SX126X_SPI_SPEED    (SPI_CLK_1MHZ)
+#define SX126X_SPI_MODE     (SPI_MODE_0)
```

Comment:
Are these defaults or hardset values?

Reply:
Just doing the same as what is done for the sx127x:

https://github.com/RIOT-OS/RIOT/blob/5e5f86fdc1ada21256a8905cd5ad8ef5489a953a/drivers/sx127x/sx127x_internal.c#L41-L42

Reply:
OK.